engine/src/gfx/mesh.cpp

```diff
@@ -125,7 +125,7 @@ bool Mesh::LoadExistant(Mesh *oldmesh) {
     try {
         *this = *oldmesh;
     } catch (std::exception& ex) {
-        VS_LOG(error, (boost::format("Error in %1%: '%2%'") % __FUNCSIG__ % ex.what()));
```

Comment:
It looks like you don't like rebases, but I'd squash/fixup this commit with the relevant commit, which is also in this same PR... But maybe that's just me, feel free to ignore.

Reply:
You are correct. I don't like rebases. 😄

Reply:
yes - if you want to rebase do so before sharing. Once shared rebase is not an option.